create-release.html.md.erb

```diff
@@ -131,7 +131,7 @@ case $1 in
 
   start)
     mkdir -p $RUN_DIR $LOG_DIR
-    chown -R vcap:vcap $RUN_DIR $LOG_DIR
+    chown vcap:vcap $RUN_DIR $LOG_DIR
```

Comment:
Unless a release wants a more restrictive/open permissions than the default agent creates?